src/api/ci/jenkins/types/jenkins.types.ts

```
@@ -0,0 +1,158 @@
+import { JenkinsBuildResult, JenkinsBuildTrigger, CredentialType } from '../enums/jenkins.enums';
```

ESLint

[error] Replace `JenkinsBuildResult,·JenkinsBuildTrigger,·CredentialType` with `CredentialType,·JenkinsBuildResult,·JenkinsBuildTrigger`

(prettier/prettier)